flict/flictlib/project/reader.py

```diff
@@ -19,8 +19,8 @@ def __init__(self, project_file, project_dirs):
 
     @staticmethod
     def dependencies_license(package):
-        licenses = [Project.package_license(dep) for dep in package.get('dependencies', [])]
-        return ' AND '.join(licenses)
+        licenses = [f' ( {Project.package_license(dep)} ) ' for dep in package.get('dependencies', [])]
+        return ' AND  '.join(licenses)
```

Comment:
This extra space doesn't sit right with me

Reply:
`        return ' AND  '.join(licenses)`
replaced by
`        return ' AND '.join(licenses)`